packages/store/src/IStore.sol

```diff
@@ -5,9 +5,9 @@ import { SchemaType } from "./Types.sol";
 import { Schema } from "./Schema.sol";
 
 interface IStore {
-  event MudStoreSetRecord(bytes32 table, bytes32[] key, bytes data);
-  event MudStoreSetField(bytes32 table, bytes32[] key, uint8 schemaIndex, bytes data);
-  event MudStoreDeleteRecord(bytes32 table, bytes32[] key);
+  event StoreSetRecord(bytes32 table, bytes32[] key, bytes data);
+  event StoreSetField(bytes32 table, bytes32[] key, uint8 schemaIndex, bytes data);
+  event StoreDeleteRecord(bytes32 table, bytes32[] key);
```

Comment:
Are these here mostly so we get them as part of the ABI?

Related to my other comment, I wonder if it would make sense to move them into a `library StoreEvents` and emit them as `emit StoreEvents.StoreSetRecord` etc.?

I don't feel that strongly about centralizing them but just trying to make sure we don't end up in a spot where we update one event signature but not the other.

Reply:
Looks like the events won't be exported in the ABI if used from a library: https://github.com/ethereum/solidity/issues/12573

Reply:
Yep that's the reason we have to declare them twice, it's unfortunate but seems like the only way to emit them in a library but make them part of a contract's ABI